packages/core/src/drizzle/kit/index.ts

```diff
@@ -45,18 +46,26 @@ export const getReorgTable = <config extends TableConfig>(
   const schema = getTableConfig(table).schema;
 
   if (schema && schema !== "public") {
-    return pgSchema(schema).table(sqlToReorgTableName(getTableName(table)), {
+    return pgSchema(schema).table(
+      sqlToReorgTableName(getTableName(table)),
+      {
+        operation_id: serial().notNull().primaryKey(),
+        operation: integer().notNull().$type<0 | 1 | 2>(),
+        checkpoint: varchar({ length: 75 }).notNull(),
+      },
+      (table) => [index().on(table.checkpoint)],
```

Comment:
Makes sense, but I would not be surprised if this has a non-trivial perf and storage impact - checkpoint strings are long.

Reply:
Hopefully this table is giant, the old values get cleared out after every finalization.